kubectl-rook-ceph.sh

```diff
@@ -557,6 +590,9 @@ function run_main_command() {
   debug)
     run_debug "$@"
     ;;
+  dr)
```

Comment:
What's the expected command? Instead of just the `dr` command, what about `dr health`?

Reply:
> Do you have readme updates for this new feature? It would help give an overview for the PR review as well.

right, I have started this pr #58 for docs improvement and will add it according to that soon

Reply:
> What's the expected command?

this is the new `kubectl rook-ceph dr health` expected command, will be better once I add the docs.

> Instead of just the `dr` command, what about `dr health`?

I don't think we can do it above format expect one word string.